docs/deployment/setup_gardener.md

```diff
@@ -1,44 +1,412 @@
 
-After preparing your values in a separate `controlplane-values.yaml` file ([values.yaml](../../charts/gardener/controlplane/values.yaml) can be used as starting point), you can run the following command against your garden cluster:
+![Gardener Setup](./content/gardener-setup.png)
+
+Reference documentation:
+- [Getting Started](https://gardener.cloud/docs/getting-started/architecture/)
+
+## Prerequisites
+
+### Runtime cluster
+
+In order to install and run Gardener an already existing Kubernetes cluster is required. It serves as the root cluster hosting all the components required for the virtual Garden cluster - like a `kube-apiserver` and `etcd` but also admission webhooks or the `gardener-controller-manager`.
+
+Reference documentation:
+
+- [Gardener Architecture](https://github.com/gardener/gardener/blob/master/docs/concepts/architecture.md)
+- [Gardener concepts](https://github.com/gardener/gardener/tree/master/docs#concepts)
+
+In this example, the runtime cluster is also used to host the very first `gardenlet`, which will register this cluster as a `Seed` as well. The `gardenlet` can be deployed to another Kubernetes cluster (recommended for productive setups). Here, using the runtime cluster keeps the amount of unmanaged clusters at a bare minimum.
+
+### DNS Zone
+
+The virtual Garden API endpoint (`kube-apiserver`) is exposed through a LoadBalancer `Service` and its IP address is made discoverable through DNS. Hence, a zone where an `A` record can be created is required.
+
+Additionally, Gardener advertises `kube-apiservers` of `Shoots` through two DNS records, which requires at least one DNS zone and credentials to interact with it through automation.
+
+Assuming the base domain for a Gardener landscape is `gardener.crazy-botany.cloud`, the following example shows a possible way to structure DNS records:
+
+| Component                | General                                            | Example                                              |
+|:-------------------------|:---------------------------------------------------|:-----------------------------------------------------|
+| Gardener Base            | gardener.\<showroom-base-domain\>                  | gardener.crazy-botany.cloud                          |
+| Virtual Garden           | api.garden.\<gardener-base-domain\>                | api.garden.gardener.crazy-botany.cloud               |
+| Gardener Ingress         | ingress.garden\<gardener-base-domain\>             | ingress.garden.gardener.crazy-botany.cloud           |
+| Gardener Dashboard       | dashboard.ingress.\<gardener-base-domain\>         | dashboard.ingress.garden.gardener.crazy-botany.cloud |
```

Comment:
Please, add also the discovery server domain. It is unconditionally deployed by GOP and used to expose public information of Gardener and Shoot clusters (CA certs and OIDC discovery metadata as of now).
```suggestion
| Gardener Dashboard       | dashboard.ingress.\<gardener-base-domain\>         | dashboard.ingress.garden.crazy-botany.gardener.cloud |
| Gardener Discovery       | discovery.ingress.\<gardener-base-domain\>         | dashboard.ingress.garden.crazy-botany.gardener.cloud |
```